### ISSUE TITLE
Cannot box ionic 2.1.0

### ISSUE THREAD

**Issue by pardeike** [arei/npmbox#63]
When I run **npmbox ionic** on OS X 10.11.6 I get

```
> npmbox ionic
Boxing ionic...
  Querying ionic
  Querying @ionic/app-generators@0.0.3
  Querying cli-table@0.3.1
  ....
  Querying source-map@~0.4.0
  Querying source-map@~0.3.0
  Downloading @ionic/app-generators@0.0.3@0.0.3
  Downloading Base64@0.2.1
  Downloading JSONStream@1.2.1
  ....
  Downloading yallist@2.0.0
  Downloading zip-stream@0.1.4

ERROR:  Error occurred downloading '@ionic/app-generators@0.0.3@0.0.3' to cache. 

npmbox halted.
```

Installed versions: npmbox 4.0.0, node 6.7.0, npm 3.10.3

**Comment by arei** [comment]
npmbox obtains the version number of a package name by splitting on the @ character.  However, with namespaced npm packages, there are two @ characters and this was confusing npmbox. 

Fixed - v4.0.1

**Comment by arei** [comment]
Thank you for the bug report!
